resources/faq/how-to-remote-access-the-web-dashboard.md

```diff
@@ -115,7 +115,7 @@ Tap on plus sign and select **New Identity**.
 
 #### New Identity
 
-Specify a name of the Identity in the field **Name**, provide a username for your PC with storagenode in the **Username** filed, specify a password in the **Password** field, confirm creation with a confirmation $$\sqrt{}$$ button.
+Specify a name of the Identity in the field **Name**, provide a username for your PC with storagenode in the **Username** filed, specify a password in the **Password** field, confirm creation.
```

Comment:
```suggestion
Specify a name of the Identity in the field **Name**, provide a username for your PC with storagenode in the **Username** field, specify a password in the **Password** field, confirm creation.
```

Reply:
I just fixed it in the next commit, so I you can just close this thread.

Reply:
I don't see any resolve conversation button below

Reply:
Seems it's related to rights. Perhaps @profclems can help you.

Reply:
I resolved this thread if you don't mind.